toolkit/tools/pkg/imagecustomizerlib/customizeselinux_test.go

```diff
@@ -206,24 +224,86 @@ func TestCustomizeImageSELinuxNoPolicy(t *testing.T) {
 	}
 }
 
-func verifyKernelCommandLine(t *testing.T, imageConnection *imageconnection.ImageConnection, existsArgs []string,
-	notExistsArgs []string,
+func verifyKernelCommandLine(t *testing.T, imageConnection *imageconnection.ImageConnection, hasUkis bool,
+	existsArgs []string, notExistsArgs []string,
 ) {
-	grubCfgFilePath := filepath.Join(imageConnection.Chroot().RootDir(), "/boot/grub2/grub.cfg")
-	grubCfgContents, err := file.Read(grubCfgFilePath)
-	assert.NoError(t, err, "read grub.cfg file")
+	var grubCfgContents string
+
+	if hasUkis {
+		// UKI image - extract cmdline from UKI files
+		ukiDir := filepath.Join(imageConnection.Chroot().RootDir(), "boot/efi/EFI/Linux")
+		cmdlineFromUki, err := extractCmdlineFromUkiForTest(ukiDir)
+		if err != nil {
+			t.Fatalf("Failed to extract cmdline from UKI: %v", err)
+			return
+		}
+		grubCfgContents = cmdlineFromUki
+	} else {
+		// GRUB image - read grub.cfg
+		grubCfgFilePath := filepath.Join(imageConnection.Chroot().RootDir(), "/boot/grub2/grub.cfg")
+		contents, err := file.Read(grubCfgFilePath)
+		if err != nil {
+			t.Fatalf("Failed to read grub.cfg: %v", err)
+			return
+		}
+		grubCfgContents = contents
+	}
 
 	for _, existsArg := range existsArgs {
-		assert.Regexpf(t, fmt.Sprintf("linux.* %s ", regexp.QuoteMeta(existsArg)), grubCfgContents,
+		assert.Regexpf(t, fmt.Sprintf("(linux.* %s |%s)", regexp.QuoteMeta(existsArg), regexp.QuoteMeta(existsArg)), grubCfgContents,
```

Comment:
Use a different regex when the args come from UKIs.

Reply:
Revised.